tmt/export.py

```diff
@@ -213,7 +223,11 @@ def create_nitrate_case(test):
         category = 'Sanity'
 
     # Create the new test case
-    summary = test.node.get('extra-summary', test.summary)
+    remote_dirname = os.path.basename(test.fmf_id['url'])
```

Comment:
The `url` can contain the `.git` suffix:

    https://github.com/psss/tmt.git

We should make sure it is stripped.

Reply:
b0242c3